spec/features/user_votes_spec.rb

```diff
@@ -20,14 +21,18 @@
 
       click_button "Log in"
 
-      visit station_path(review.station)
+      visit station_path(@review.station)
     end
 
-    scenario 'User votes on review' do
-      # figure out voting mechanism on form
+    scenario 'User votes on review', focus: true do
       click_link "up"
 
       expect(page).to have_content("Thanks for your vote")
+      expect(ActionMailer::Base.deliveries.size).to eql(1)
+
+      last_email = ActionMailer::Base.deliveries.last
+      expect(last_email).to have_subject('Your review has a new vote!')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.